integration_test.sh

```diff
@@ -1,18 +1,16 @@
 set -e
```

Comment:
This is my integration testing tool I run before committing.
It sends files all around and diffs them.
I want to move all this into Docker because it leaves a couple folders and you must delete the projects via the portal.